man/rules/meson.build

```diff
@@ -699,6 +699,7 @@ manpages = [
  ['systemd.socket', '5', [], ''],
  ['systemd.special', '7', [], ''],
  ['systemd.swap', '5', [], ''],
+ ['systemd.syntax', '7', [], ''],
```

Comment:
Did the new file not get added?

Reply:
Indeed, somehow. I distinctly remember calling `git add -N` (it's necessary for `update-man-rules` target to work), strange. Updated.